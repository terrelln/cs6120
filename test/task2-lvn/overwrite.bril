@main() {
    b: int = const 1;
    b: int = const 1;
    b: int = const 1;
    print b;
}