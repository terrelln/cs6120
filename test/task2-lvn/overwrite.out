@main {
._block0:
  __var0: int = const 1;
  print __var0;
  ret;
}
